src/decode-events.c

```diff
@@ -266,10 +266,6 @@ const struct DecodeEvents_ DEvents[] = {
             "decoder.udp.hlen_too_small",
             UDP_HLEN_TOO_SMALL,
     },
-    {
```

Comment:
how will this work wrt loading an old rule? Test with `rules/decoder-events.rules`

Reply:
Good catch, it fails to start and complains about using an invalid keyword. 
I can create a deprecation notice for the keyword in 7.0.0 and silently ignore the keyword in 6.0.9?